### ISSUE TITLE
Adding more sources

### ISSUE THREAD

**Issue by Niquarl** [milesmcc/LibreNews-Android#26]
Would it be possible to add more sources. The BBC's alerts are great but I think they are pretty UK and USA focussed.

**Comment by milesmcc** [comment]
I agree wholeheartedly. I'm currently in the process of working with other news agencies to launch their own LibreNews instances.

(BBC is the source right now because they are they only news stream that _isn't_ incredibly spammy. It'd be such a pain to get notifications every five minutes, so I've avoided using streams from CNNBreaking, NYTimes, and RT, for example.)

**Comment by Niquarl** [comment]
Ok thanks for the reply. This is great. Greatly looking forward to that. I'm assuming they are all English speaking sources ?

-------- Original Message --------

**Comment by Serkan-devel** [comment]
Maybe Reuters could be added

**Comment by milesmcc** [comment]
@Serkan-devel that would be interesting, but does Reuters offer a non-spammy breaking news feed (4-8 notifications per day, max).

@Niquarl the data is encoded UTF-8, so any number of languages are supported. The beauty of LibreNews' design is that it is decentralized; that is, anyone could start a LibreNews instance for their particular language/region.

**Comment by Niquarl** [comment]
Ok, I'll look into putting up my own instance maybe then. I'll go and check out the other repo.

**Comment by tp0** [comment]
> @Serkan-devel that would be interesting, but does Reuters offer a non-spammy breaking news feed (4-8 notifications per day, max).

Reuters' "Top News" RSS feed is fairly clean, but still has more than 4-8 per day.

http://feeds.reuters.com/reuters/topNews

https://www.reuters.com/tools/rss

**Comment by Serkan-devel** [comment]
This API requires registration, but is free to use in their TOS when credited. I can help if you want to contact them

https://api.presseportal.de/demo/

**Comment by jomo** [comment]
Hey all, I'm running a LibreNews server at https://librenews.jomo.tv (use `https://librenews.jomo.tv/api` as server URL).

Currently it provides a bunch of international and some German news feeds. The sources are basically the Twitter accounts followed by [@librenewz](https://twitter.com/librenewz), but I'm happy to add more (Twitter) sources upon request.

I had to remove [@reuters](https://twitter.com/reuters) though, they post a little too often on that account.

Please note each source is in a separate channel, so you can pick which sources you want to follow, but you'll need to wait for #32 to be merged in or use something like [Preference Manager](https://f-droid.org/packages/fr.simon.marquis.preferencesmanager/) to manually configure the channels you want to subscribe.

**Comment by HonestAbe7** [comment]
Adding to this, what about adding Fox News?
